src/vignette/util/thumbnail.clj

```diff
@@ -74,6 +74,7 @@
                (file-exists? temp-file))) (io/file temp-file)
       :else (throw+ {:type :convert-error
                      :error-code (:exit sh-out)
+                     :out-string (:out sh-out)
```

Comment:
I think we used to have this, but I removed it because there is a limit to the size of the message the java library will send using the syslog protocol. When it goes over, the json is partially missing, so it can't be json decoded, and the message is effectively lost (though the raw json string does make it to ES)